src/drivers/differential_pressure/ms4525/ms4525_airspeed.cpp

```diff
@@ -206,25 +202,18 @@ MEASAirspeed::collect()
 	  positive number when the top port is used as a dynamic port
 	  and bottom port is used as the static port
 	 */
-
-	if (PX4_ISFINITE(diff_press_pa_raw)) {
-		differential_pressure_s report{};
-
-		report.error_count = perf_event_count(_comms_errors);
-		report.temperature = temperature;
-		report.differential_pressure_filtered_pa = _filter.apply(diff_press_pa_raw) - _diff_pres_offset;
-		report.differential_pressure_raw_pa = diff_press_pa_raw - _diff_pres_offset;
-		report.device_id = _device_id.devid;
-		report.timestamp = hrt_absolute_time();
-
-		_airspeed_pub.publish(report);
-	}
-
-	ret = OK;
+	differential_pressure_s report;
+	report.timestamp_sample = timestamp_sample;
+	report.device_id = get_device_id();
+	report.differential_pressure_pa = diff_press_pa_raw;
```

Comment:
I guess we can drop now the "raw" in `diff_press_pa_raw`?

Reply:
Thanks, fixed.